### PR TITLE
don't reuse support graph ids

### PR DESCRIPTION
**Pull request by cbizon**

### CODE REVIEW

File: ranker/modules/omnicorp_overlay.py [hunk 1]
```diff
@@ -94,6 +93,7 @@ async def add_shared_pmid_counts(
                     omnisupport = sg
```
Comment (maximusunc) [inline ranker/modules/omnicorp_overlay.py:93]: Can there only ever be one omnicorp support graph per result?

**Review by maximusunc — APPROVED** [review]
Just had one question, but this looks good

**Review by uhbrar — APPROVED** [review]
This must have gotten lost in the shuffle, but it looks good to me.

File: ranker/modules/omnicorp_overlay.py (hunk 1 above)
Comment (maximusunc) [reply ranker/modules/omnicorp_overlay.py:93]: If so, though marginal, we should break out of this loop once we find it.

Comment (cbizon) [reply ranker/modules/omnicorp_overlay.py:93]: Good point

**Pull request merged**